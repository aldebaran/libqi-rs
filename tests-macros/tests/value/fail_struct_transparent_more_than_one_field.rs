#[derive(qi_macros::Reflect, qi_macros::ToValue, qi_macros::IntoValue, qi_macros::FromValue)]
#[qi(value(crate = "qi_value", transparent))]
struct S {
    a: i32,
    b: i32,
}

fn main() {}
