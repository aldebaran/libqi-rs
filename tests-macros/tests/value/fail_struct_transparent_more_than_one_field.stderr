error: `qi(transparent)` requires struct to have at most one field
 --> tests/value/fail_struct_transparent_more_than_one_field.rs:3:8
  |
3 | struct S {
  |        ^
